### PR TITLE
Add option to load the dashboards via the Kibana API 

### PR DESCRIPTION
**Pull request by monicasarbu**
Trying to solve https://github.com/elastic/beats/issues/4712

This PR adds the option to the `import_dashboards` script to load the dashboards using the Kibana API. By default, the `import_dashboards` loads the dashboards to Elasticsearch. The same behavior has the `-setup` option.

This PR is a manual backport from master and it's customized for 5.6 branch. The motivation is that loading the dashboards in Beats 5.x to work with Elasticsearch 6.x version.

### CODE REVIEW

File: libbeat/dashboards/import_dashboards.go [hunk 1]
```diff
@@ -122,20 +120,28 @@ func (cl *CommandLine) ParseCommandLine() error {
...
 	}
 	/* parse command line arguments */
 	err = cl.ParseCommandLine()
 	if err != nil {
-		return nil, err
+		return err
 	}
 
-	cfg := dashboards.DashboardsConfig{
+	/* prepare the Elasticsearch index pattern */
```
Comment (ruflin) [inline libbeat/dashboards/import_dashboards.go]: leftover?

File: libbeat/dashboards/import_dashboards.go [hunk 2]
```diff
@@ -149,50 +155,75 @@ func New() (*dashboards.Importer, error) {
...
 	}
 
-	/* prepare the Elasticsearch index pattern */
-	fmtstr, err := fmtstr.CompileEvent(cl.opt.Index)
-	if err != nil {
-		return nil, fmt.Errorf("Failed to build the Elasticsearch index pattern: %s", err)
-	}
-	indexSel := outil.MakeSelector(outil.FmtSelectorExpr(fmtstr, ""))
+	/* TLS config */
+	/*
```
Comment (ruflin) [inline libbeat/dashboards/import_dashboards.go]: leftover?

File: libbeat/docs/version.asciidoc [hunk 3]
```diff
@@ -1,4 +1,4 @@
-:stack-version: 5.5.0
-:doc-branch: 5.5
+:stack-version: 5.6.0
```
Comment (ruflin) [inline libbeat/docs/version.asciidoc]: @monicasarbu did you run set-version for this?
@tsg Kind of concerned that we miss that somewhere else.

Comment (tsg) [inline libbeat/docs/version.asciidoc]: That's not set by `set_version`, but it's part of the release checklist to update it. I think it's fine to be changed here.

**Comment by tsg** [comment]
jenkins, package it

**Comment by tsg** [comment]
Tested manually with 6.0.0-beta1 and 5.4.1. Both seem to work fine. Nice work, @monicasarbu!

**Pull request merged**